### DISCUSSION TITLE
v5.2.3

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16385] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.2.3'>v5.2.3</a>.</em>

**Comment by s-ohnishi** [comment]
I've only tried it a little, but it seems fine so far without the `self` problem unless you use `Phalcon\Tag`.
As said, using Select in Forms was a no-go because of the `self` problem.
It seems that the checkbox and radio are not according to the document (following the Tag document, not the TagFactory), but so far there have been no `self` problems.
